CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to this project will be documented in this file - [read more
 
 ### Fixed
 - Error when a subclassed integration returns an object that cannot be cast as a string #423
+- initial redis auto instrumentation integration (https://pecl.php.net/package/redis)
```

Comment:
I would move this under the `## Added` header and change the copy to the following:

```suggestion
- Support for [ext/redis](https://pecl.php.net/package/redis) #428
```